oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/NodeDocumentTest.java

```diff
@@ -984,7 +984,7 @@ public <T extends Document> T find(Collection<T> collection,
             prevDocCalls.clear();
             String value = doc.getVisibleChanges("p", headRevisions.get(i), null).iterator().next().getValue();
             assertEquals(String.valueOf(numChanges - (i + 1)), value);
-            assertTrue("too many calls for previous documents: " + prevDocCalls,
+            assertTrue("too many calls for previous documents: " + prevDocCalls + " : " + prevDocCalls.size(),
```

Comment:
This change seems unrelated.

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/2049/commits/71020cc5ef9c154589a554c5ab83d68dd4187b80